include/game.h

```diff
@@ -686,6 +686,20 @@ typedef struct {
     /* 0x12 */ u16 sp22; // entity->objectRoomIndex
 } SubweaponDef;          /* size=0x14 */
 
+typedef enum {
+    ITEM_SSWORD,
+    ITEM_SWORD,
+    ITEM_THROW1,
```

Comment:
```suggestion
ITEM_THROW_1

Reply:
Thoughts then on SSWORD versus S_SWORD?

Reply:
Not sure that's the difference between S_SWORD and SWORD, but yeah i think the underscore should be there.